### PR TITLE
Initialize the project for development

### PR DESCRIPTION
**Pull request by Gravewalker666**
## Purpose

To initialize the project for development

## Approach

- Install and configure tailwind CSS
- Make the community developer webpage
- Setup JSON data file for contributor data
- Add pull requests and issue templates

## Screenshots
image

## Environment
- Safari, Google Chrome
- macOS Big Sur

### CODE REVIEW

File: getDevelopers.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+        type: 'get',
+        url: 'developers.json',
+        dataType: 'json',
+        success: function (data) {
+            console.log(data)
+            let templateDev = Mustache.render($("#dev-template").html(), {"data": data});
+            $("#wrapper").html(templateDev);
+        }
+    });
+})
```
Comment (anjula-sack) [inline getDevelopers.js:12]: EOL

File: getDevelopers.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+$(function () {
+    $.ajax({
+        type: 'get',
+        url: 'developers.json',
+        dataType: 'json',
+        success: function (data) {
+            console.log(data)
```
Comment (anjula-sack) [inline getDevelopers.js:7]: Let's remove the console.log

**Review by anjula-sack — CHANGES_REQUESTED** [review]
Good job! Small changes

File: getDevelopers.js (hunk 2 above)
Comment (Gravewalker666) [reply getDevelopers.js:7]: ah right away, 👍

File: getDevelopers.js (hunk 1 above)
Comment (Gravewalker666) [reply getDevelopers.js:12]: sure, on it

**Comment by Gravewalker666** [comment]
@anjula-sack 
Requested changes made with commit 7548f34 please review 🙂

**Review by anjula-sack — APPROVED** [review]
Cool design. Thank you! @Gravewalker666

**Pull request merged**